registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -584,6 +587,7 @@
     "sha256:2aabec90cdb8b65bfd5cd71842c3f7a04fbe0b702fd2faea5bcae6c882f7e8e6": ["v1.1.4"]
     "sha256:481bb3891b4cb48912b4d0822333f3bd23429b3eb2f0293770fc249de647102b": ["v1.1.5"]
     "sha256:f7abd1a99c5aabb8724f9ac59217900835578a8dc498ea54be00c1fd9cc40aef": ["v1.1.6"]
+    "sha256:dd6aadab83f675eba420242a99e310b0d5ae2dad77d71df37161cad19282ceac": ["v1.10.0"]
```

Comment:
`dd6aadab83f6` matches the SHA for kubeadm-control-plane-controller v1.10.0 in the [staging repository](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-control-plane-controller@sha256:dd6aadab83f675eba420242a99e310b0d5ae2dad77d71df37161cad19282ceac/details). 👍🏻